certbot/certbot/display/ops.py

```diff
@@ -19,58 +19,31 @@
 logger = logging.getLogger(__name__)
 
 
-def get_email(invalid: bool = False, optional: bool = True) -> str:
+def get_email(invalid: bool = False) -> Optional[str]:
```

Comment:
Since we're changing the return value to allow `None` and this is in our public API... we need to do a major release to push this change. Luckily, we have one of those planned. We may also therefore want to do a release warning that the change is going to happen, before that.

Reply:
Probably a bit, but either way is fine.

Reply:
Cool, I reverted to keep the old signature.